packages/lodestar/src/network/network.ts

```diff
@@ -152,9 +155,14 @@ export class Network implements INetwork {
   /**
    * Request att subnets up `toSlot`. Network will ensure to mantain some peers for each
    */
-  prepareBeaconCommitteeSubnet(subscriptions: phase0.BeaconCommitteeSubscription[]): void {
-    this.attService.addBeaconCommitteeSubscriptions(subscriptions);
-    this.peerManager.onBeaconCommitteeSubscriptions();
+  prepareBeaconCommitteeSubnet(subscriptions: CommitteeSubscription[]): void {
+    this.attnetsService.addCommitteeSubscriptions(subscriptions);
+    if (subscriptions.length > 0) this.peerManager.onCommitteeSubscriptions();
+  }
+
+  prepareSyncCommitteeSubnets(subscriptions: CommitteeSubscription[]): void {
+    this.syncnetsService.addCommitteeSubscriptions(subscriptions);
```

Comment:
I suppose this is not the same to `prepareBeaconCommitteeSubnet`? the subnet of SyncCommittee is totally different from attestation so it's not the same business logic there.

Instead, we only need to use gossip to subscribe to sync committee subnet here?!

Reply:
No, it's not the same. Note that there are two different classes
- `attnetsService`: Manage attestation subnets
- `syncnetsService`: Manage sync committee subnets

Both instantiate the same class `SubnetsService` since the logic to handle attnets and syncnets is *almost* identical. I've created this generic `CommitteeSubscription` type that gives just the minimal info needed for `SubnetsService`